### DISCUSSION TITLE
Vesoft Hackathon 2022 流水账 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#11] [Comments]
# Vesoft Hackathon 2022 流水账 - siwei.io

NebulaGraph 终于有 Fuzz 测试工具了，NebulaGraph 上跑向量搜索，基于 Kubesphere 的多集群 PaaS。

https://siwei.io/vesoft-hackathon/

**Comment by gaocegege** [comment]
很酷

**Reply by wey-gu** [reply]
谢谢 gaoce 哥哥♥️
